### PR TITLE
Filters Works Separately

### PR DESCRIPTION
**Pull request by marianbme**

### CODE REVIEW

File: StreetCare/StreetCare/Views/Screens/CommunityEvents/CommunityEventView.swift [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+  
+    var body: some View {
+        
+        VStack {
+            HStack {
+                // Search Bar on the left
+                Image(systemName: "magnifyingglass")
+                    .padding(.horizontal, 10)
+                    .foregroundColor(.black)
+                TextField("Search...", text: $viewModel.searchText)
```
Comment (ameykanunje) [inline StreetCare/StreetCare/Views/Screens/CommunityEvents/CommunityEventView.swift:32]: Text should be using Localized strings

File: StreetCare/StreetCare/Views/Screens/CommunityEvents/EventCardView.swift [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                    .background(Color("Color87CEEB").opacity(0.4))
+                    .cornerRadius(5)
+            }
+            
+            HStack {
+                if let slots = event.event.totalSlots {
+                    let minimumInterest = Int(Double(slots) * 0.65)
+                    let interest = Int.random(in: minimumInterest...slots)
+
+                    Text("Participants: \(interest) / \(slots)")
```
Comment (ameykanunje) [inline StreetCare/StreetCare/Views/Screens/CommunityEvents/EventCardView.swift:49]: Text in localized format

File: StreetCare/StreetCare/Views/Screens/CommunityEvents/EventCardView.swift [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                        .font(.system(size: 13))
+                }
+                Spacer()
+                if eventType == .past {
+                    Text("Completed").font(.system(size: 13))
```
Comment (ameykanunje) [inline StreetCare/StreetCare/Views/Screens/CommunityEvents/EventCardView.swift:54]: Localized String

File: StreetCare/StreetCare/Views/Screens/CommunityEvents/CommunityEventView.swift [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+            
+            if let _ = self.user {
+                if viewModel.filteredData.isEmpty {
+                    Spacer(minLength: 50.0)
+                } else {
+                    List {
+                        ForEach(0..<viewModel.filteredData.count, id: \.self) { index in
+                            if let date = viewModel.filteredData[index].keys.first,
+                               let eventObj = viewModel.filteredData[index][date] {
+                                Section(header: SectionHeaderView(date: date)) {
```
Comment (ameykanunje) [inline StreetCare/StreetCare/Views/Screens/CommunityEvents/CommunityEventView.swift:92]: Is it possible to add the count of that particular month. Just asking which will also help to show that after filtering the data, how many events are available.

**Review by ameykanunje — CHANGES_REQUESTED** [review]
Localized String
When searching for an event the participation count changes

**Comment by marianbme** [comment]
1.Completed localized strings
2.Added count for every month
3.The participant count is supposed to be commented in another PR.

**Review by phillipsk — APPROVED** [review]

File: StreetCare/StreetCare/Views/Screens/CommunityEvents/CommunityEventView.swift (hunk 4 above)
Comment (phillipsk) [reply StreetCare/StreetCare/Views/Screens/CommunityEvents/CommunityEventView.swift]: We can do a fast follow and resolve in the next deployment

**Review by phillipsk — APPROVED** [review]

**Review by ameykanunje — APPROVED** [review]

**Pull request merged**